runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -679,6 +682,54 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 		stringStack[stringStackSize - 1] += titleBuilder.toString();
 	}
 
+	private void setLimitReset(int itemId)
+	{
+		String lastLimit = getLimitReset(itemId);
+		if (!lastLimit.equals("None"))
+		{
+			return; // do not update if previous limit has not expired
+		}
+		String itemIdStr = Integer.toString(itemId);
+		Instant currentDateTime = Instant.now();
+		configManager.setConfiguration("grandexchange",
+			client.getUsername().toLowerCase() + itemIdStr, currentDateTime);
```

Comment:
I'd prefix the key with something like `buylimit_` so it doesn't clash in the future with something we might add.

Reply:
previously the key was "gelimitreset", and you requested a change to "grandexchange". Can i confirm you now want it changed to a unique identifier? If yes, i will change it to "buylimit_grandexchange"

Reply:
Previously the *group* was `gelimitreset`. The method is `setConfiguration(group, key, value)`. The group should be `grandexchange`, but the key should be prefixed. So eg `setConfiguration("grandexchange", "buylimit_foo@bar.com_4151", instant)`

Reply:
Sorry, misread/misunderstood. I'll change it!